python/examples/installing-a-package.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/python3
+
+# Install a package from a file stored on your system.
+# A Python equivalent for `rpm -i hello-2.12.1-4.fc40.x86_64.rpm`
+
+import os
+import rpm
+
+ts = rpm.TransactionSet()
+path = "/tmp/hello-2.12.1-4.fc40.x86_64.rpm"
+with open(path, "r") as fp:
+    filename = os.path.basename(path)
+    hdr = ts.hdrFromFdno(fp.fileno())
+    ts.addInstall(hdr, filename, "i")
+ts.run(lambda *args: print(args), {})
```

Comment:
I have never actually installed anything through the Python API in real life, so I am not sure what the `ts.run` arguments should be. Same for the erasing example above.

Also, this script fails with

```
# FIXME this snippet fails with
TypeError: 'NoneType' object cannot be interpreted as an integer
FATAL ERROR: python callback ??? failed, aborting!
```

Can you please help me finish this example?

Reply:
That said, the callback example actually sets a rather bad example: you don't want to use a header in the key, because it wastes enormous amounts of memory. If you want output a NEVRA in the callback, pass that string instead.